pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -394,21 +412,84 @@ func (c *Cluster) startProvisioningOverNodes(config *provisionConfig) {
 			storeConfig:    storeConfig,
 			metadataDevice: metadataDevice,
 		}
-		job, err := c.makeJob(osdProps, config)
+		c.makeAndRunJob(n.Name, "provision", osdProps, config)
+	}
+}
+
+func (c *Cluster) startNodeDriveGroupProvisioners(config *provisionConfig) {
+	logger.Debug("starting provisioning on nodes using Drive Groups config")
+
+	if c.DesiredStorage.UseAllNodes {
```

Comment:
These warnings wouldn't be needed if we simply don't support the mixed mode.

Reply:
We don't support the mixed mode, and my intention is that these warnings help make that more clear to users since they warn against the two being configured at the same time. Or would you suggest throwing an error at this point and forcing the user to choose one or the other?

Reply:
Actually, I would suggest we add these checks in the admission controller so they wouldn't be needed here. As a separate work item, I still want to get to the point where the admission controller is always enabled.